lib/12T/.vscode/configurationCache.log

```diff
@@ -0,0 +1 @@
+{"buildTargets":[],"launchTargets":[],"customConfigurationProvider":{"workspaceBrowse":{"browsePath":[]},"fileIndex":[]}}
```

Comment:
gitignore .vscode? (so that it doesn't get accidently commited)

Reply:
not needed.  Deleted.